holoviews/ipython/__init__.py

```diff
@@ -194,7 +194,7 @@ def __call__(self, *args, **params):
                                                   "hv-extension-comm")
 
         # Create a message for the logo (if shown)
-        if not same_cell_execution:
+        if not same_cell_execution and p.logo:
             self.load_hvjs(logo=p.logo,
```

Comment:
Given that panel handles the JS now, I guess this is now misnamed (an artifact of history). Could you double check it is only used to load HTML and if so, rename it to `self.load_extension_html` (or similar)?